toolkits/global/packages/global-forms/README.md

```diff
@@ -13,3 +15,265 @@ The `global-forms` component currently uses the `DEFAULT` brand only.
+
+```
+"fieldsets": {
+    [
+        {
+            "fields": [
+                {
+                    "type": "text",
+                    "label": "Your name",
+                    "id": "your-name",
+                    "name": "your-name"
+                },
+                {
+                    "type": "text",
+                    "label": "Your email",
+                    "id": "your-email",
+                    "name": "your-email"
+                }
+            ]
+        }
+    ]
+}
+```
+
+Where you do wish to include a legend, HTML is permissable, meaning you can include a heading to reinforce the form and page structure:
+
+```
+"fieldsets": {
+    [
+        {
+            "legend": "<h2>My level 2 legend</h2>",
+            "fields": [
+                ...
+            ]
+        }
+    ]
+}
+```
+
+(**Note:** it is valid and conforming to use heading elements inside `<legend>`s.)
+
+### Fields
+
+Fields take the `type` property to determine which of the `view/fields` they represent. This property along with `label`, `id`, and `name` should be mandatory in terms of your model.
+
+A wide range of standard input/field attributes are supported. So, for example, if you wanted to include a `readonly` attribute on your text input, you would just include a property of the same name on the data:
+
+```
+{
+    "type": "text",
+    "label": "Your email",
+    "id": "your-email",
+    "name": "your-email",
+    "readonly": true
+}
+```
+
+The `hint` property includes supplementary text under the main `<label>` text but _inside_ the `<label>`, meaning it is automatically available to screen reader software.
+
+```
+"hint": "This will be the email you signed up using"
+```
+
+The `optional` property affects the field’s label, appending _“(optional)”_ to the label text.
+
+```
+"optional": true
+```
+
+### Errors
+
+Each field can have an `error` property. The presence of the property indicates the field is in an error state and the property value (a string) defines the error message.
+
+```
+{
+    "type": "text",
+    "label": "Your name",
+    "id": "your-name",
+    "name": "your-name",
+    "error": "That is not a name, it’s a number"
+}
+```
+
+Errors can be summarized using a top level `errorSummary` property (adjacent to the `fieldset` property). Each error in the errors array must point to the `id` of the respective input and repeat its `error` message:
+
+```
+"errorSummary": {
+    "id": "summary",
+    "title": "Here’s what’s wrong",
+    "errors": [
+        {
+            "id": "your-name-error",
+            "error": "That is not a name, it’s a number"
+        },
+        {
+            "id": "checkbox-terms-error",
+            "error": "You must agree to the terms!"
+        }
+    ]
+}
+```
+
+### Multiple choices
```

Comment:
Hold on, radios and select (unless using `multiple` attribute) are not multiple choices. Do we support the `select[multiple]` so far?
Shall we host these in a different section?

Reply:
@morgaan Terminology clash. I mean each of these elements represent multiple choices (e.g. options for selects).

Reply:
Oh I see, well I'm sure @amyhupe could help there. As a non-native speaker I won't be of any help here I'm afraid. Lot of luck in finding better copy.

Reply:
💯 just as well as in general when under 5-6 options (Burn you select tags talk resonate in my head)

Reply:
This kind of guideline should make it to the doc actually in "When to use..." or "When not to use..."